bindings/Python/py11File.h

```diff
@@ -121,11 +126,13 @@ class File
 private:
     std::shared_ptr<core::Stream> m_Stream;
     adios2::Mode ToMode(const std::string mode) const;
+    static Layout ToLayout(std::string order);
```

Comment:
See my comment about `static` and changing the signature to `Layout File::ToLayout( const std::string order) const`